packages/account/imAccount/validator/P256Owner.ts

```diff
@@ -0,0 +1,95 @@
+import { p256 } from "@noble/curves/p256"
+import * as ethers from "ethers"
+
+import byte from "~packages/util/byte"
+import type { BytesLike, HexString } from "~typing"
+
+import type { WebAuthnValidatorOwner } from "../validator"
+
+export class P256Owner implements WebAuthnValidatorOwner {
```

Comment:
Since this `P256Owner` (and potentially future `webAuthnOwner`) isn't a Validator but rather intended for use by `webAuthnValidator`, should we consider moving `P256Owner.ts` to `packages/account/imAccount/validator/webAuthnValidatorOwner/`? What do you think, @pilagod ?

Reply:
The ultimate goal is to generalize passkey owner like p256 or webauthn, to be used in both `PasskeyAccount` and `WebAuthnValidator`. So we can just leave it for now, reorganizing it in the future PR.